addons/survey/views/survey_templates_statistics.xml

```diff
@@ -525,17 +539,21 @@
     </template>
 
     <template id="question_table_pagination" name="Survey: statistics table pagination">
-        <ul t-att-id="'pagination_%d' % question.id" class="pagination mt-2" t-att-data-question_id="question.id"
+        <t t-set="non_skipped_answers_count" t-value="len(question_data['answer_input_done_ids'])"/>
+        <t t-if="non_skipped_answers_count > page_record_limit">
+            <div class="d-flex justify-content-between d-print-none pagination_wrapper"
+            t-att-id="'pagination_%d' % question.id" t-att-data-question_id="question.id" 
             t-att-data-record_limit="page_record_limit">
-            <t t-set="non_skipped_answers_count" t-value="len(question_data['answer_input_done_ids'])"/>
-            <t t-if="non_skipped_answers_count > page_record_limit">
-                <t t-set="total" t-value="ceil(non_skipped_answers_count / page_record_limit) + 1"/>
-                <li t-foreach="range(1, total)" t-as="num"
-                    t-att-class="'page-item o_survey_js_results_pagination %s' % ('active' if num == 1 else '')">
-                    <a href="#" class="page-link" t-esc="num"></a>
-                </li>
-            </t>
-        </ul>
+                <ul  class="pagination mt-2">
```

Comment:
Greening the task, but there is a double space here between the tag and "class" 👀